queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM vet_clinic WHERE name LIKE '%mon%';
```

Comment:
- [ ] This query suppose to find all animals that its name ends in "mon", not all animals that have "mon" in its name. So please fix that.
[Hint] It will be like that instead:
```sql
SELECT * FROM animals WHERE name LIKE '%mon';
```